README.md

```diff
@@ -0,0 +1,26 @@
+# Lissi Connect through NodeJS
+
+This express application presents a working example of a Single Sign-On method through lissi-Connect using `passportJS` and `openid-client` strategy.
+
+## Quickstart
+
+Clone this repository using `git clone https://github.com/lissi-id/lissi-oidc-nodejs-sample.git`
```

Comment:
Please remove all reverences of `oidc`

Reply:
This is the repository name. Should I modify it as well?

Reply:
Yes, maybe something like `lissi-connect-nodejs-sample`?